### PR TITLE
Hotfix/v1.0.1: Hotfix in Deployment

### PR DESCRIPTION
**Pull request by waji46**
Title : Hotfix in Deployment

    [P.S]
    - Updated the  Readme file
    - Update the Architect Diagram in Readme file
    - Update the user-data script for Ec2
    - Update the Roadmap file
    - Added the changelog file

### CODE REVIEW

**Review by yaseenzafar123 — APPROVED** [review]
LGTM

**Review by samanXgrid — APPROVED** [review]
LGTM

**Review by muhdsaad — APPROVED** [review]
LGTM

File: infrastructure/main.tf [hunk 1]
```diff
@@ -50,6 +50,7 @@ module "xc3" {
   region              = var.region
   prometheus_layer    = var.prometheus_layer
   domain_name         = var.domain_name
+  parent_domain_name  = var.parent_domain_name
```
Comment (samanXgrid) [inline infrastructure/main.tf:53]: so do we need to remove parent_domain_name? @faheem-khau9

**Review by samanXgrid — COMMENTED** [review]
Once it is reviewed by other I will approve it as I am not sure parent domain variable is needed or not

**Review by samanXgrid — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# Changelog
```
Comment (faheem-khau9) [inline CHANGELOG.md:1]: Please merge your hotfixes into develop first

**Review by faheem-khau9 — CHANGES_REQUESTED** [review]
All the hotfixes should be merged into develop.

**Comment by faheem-khau9** [comment]
:robot: OpenAI








---

### Chat with 🤖 OpenAI Bot (`@openai`)
- Reply on review comments left by this bot to ask follow-up questions. A review comment is a comment on a diff or a file.
- Invite the bot into a review comment chain by tagging `@openai` in a reply.

### Code suggestions
- The bot may make code suggestions, but please review them carefully before committing since the line number ranges may be misaligned. 
- You can edit the comment made by the bot and manually tweak the suggestion if it is slightly off.

### Ignoring further reviews
- Type `@openai: ignore` anywhere in the PR description to ignore further reviews from the bot.

---






<details>
<summary>Files not summarized due to errors (21)</summary>

### Failed to summarize

* ROADMAP.md (nothing obtained from openai)
* CHANGELOG.md (nothing obtained from openai)
* README.md (nothing obtained from openai)
* infrastructure/config.sh (nothing obtained from openai)
* infrastructure/modules/serverless/cloudtrail.tf (nothing obtained from openai)
* infrastructure/init.sh (nothing obtained from openai)
* infrastructure/main.tf (nothing obtained from openai)
* infrastructure/init_delete.sh (nothing obtained from openai)
* infrastructure/modules/serverless/variables.tf (nothing obtained from openai)
* infrastructure/modules/serverless/data.tf (nothing obtained from openai)
* infrastructure/modules/xc3/cognito.tf (nothing obtained from openai)
* infrastructure/modules/xc3/README.md (nothing obtained from openai)
* infrastructure/modules/xc3/loadbalancer.tf (nothing obtained from openai)
* infrastructure/modules/xc3/main.tf (nothing obtained from openai)
* infrastructure/modules/xc3/output.tf (nothing obtained from openai)
* infrastructure/modules/xc3/startup-script.sh.tpl (nothing obtained from openai)
* infrastructure/terraform.test.tfvars (nothing obtained from openai)
* infrastructure/output.tf (nothing obtained from openai)
* infrastructure/terraform.auto.tfvars (nothing obtained from openai)
* infrastructure/modules/xc3/variables.tf (nothing obtained from openai)
* infrastructure/variables.tf (nothing obtained from openai)

</details>


---
In the recent run, only the files that changed from the `base` of the PR and between `33b16f740e1dff4841e641f9e4405ff8e795f7c2` and `7b2fa9d7686da303324572e056162811a7ecb90d` commits were reviewed.

<details>
<summary>Files not reviewed due to errors in the recent run (21)</summary>

### Failed to review in the last run

* CHANGELOG.md (no response)
* ROADMAP.md (no response)
* README.md (no response)
* infrastructure/config.sh (no response)
* infrastructure/init_delete.sh (no response)
* infrastructure/main.tf (no response)
* infrastructure/modules/serverless/cloudtrail.tf (no response)
* infrastructure/init.sh (no response)
* infrastructure/modules/xc3/README.md (no response)
* infrastructure/modules/serverless/data.tf (no response)
* infrastructure/modules/serverless/variables.tf (no response)
* infrastructure/modules/xc3/cognito.tf (no response)
* infrastructure/modules/xc3/loadbalancer.tf (no response)
* infrastructure/modules/xc3/output.tf (no response)
* infrastructure/modules/xc3/startup-script.sh.tpl (no response)
* infrastructure/modules/xc3/main.tf (no response)
* infrastructure/output.tf (no response)
* infrastructure/modules/xc3/variables.tf (no response)
* infrastructure/terraform.test.tfvars (no response)
* infrastructure/terraform.auto.tfvars (no response)
* infrastructure/variables.tf (no response)

</details>

**Review by faheem-khau9 — APPROVED** [review]
LGTM

**Review by yaseenzafar123 — DISMISSED** [review]
LGTM

**Review by samanXgrid — DISMISSED** [review]
LGTM

**Review by faheem-khau9 — APPROVED** [review]
LGTM

**Review by muhdsaad — APPROVED** [review]

**Review by yaseenzafar123 — APPROVED** [review]
LGTM

**Review by samanXgrid — APPROVED** [review]
LGTM

**Pull request merged**